rofi-confluence-jira.py

```diff
@@ -28,7 +28,7 @@ def fetch(self, limit=500, start=0):
             space = page['_expandable']['space'].split('/')[-1]
             title = page['title']
             url = page['_links']['webui']
-            page['url'] = f'{self.url}/{url}'
+            page['url'] = f'{self.url}/{url}'.replace('//', '/')
```

Comment:
I dislike replacing all `//` since it is unclear where those come from when just reading the code. What about using `strip` (or `lstrip`) as follows:
```py
url = page['_links']['webui'].strip('/')

# example
>>> '/foo/bar/baz'.strip('/')
'foo/bar/baz'
```

Reply:
Ah of course strip will do it much better It comes from `url` (so the API response) starting with a trailing `/` and the `/` inside the fstring results in `//`

Will adjust it on Monday